portfolio/src/main/webapp/script.js

```diff
@@ -39,3 +39,20 @@ function addRandomFact() {
   factContainer.innerHTML = '';
   factContainer.appendChild(para);
 }
+
+async function getCommentFromFetch() {
+    console.log("Try fetch");
+    const response = await fetch("/data");
+    const comments = await response.json();
+    const commentContainer = document.getElementById("comment-container")
+    commentContainer.innerHTML = "";
+    var index;
+    for (index = 0; index < comments.length; index++) {
```

Comment:
You can move the `var` declaration to the `for`-loop: `for (var index = 0; ...)`.

Even better, you can omit the `index` variable and say `for (const comment of coments) {`.

Reply:
Oh I didn't know that js supports for each loop, just like java. I will change this in the next commit.